kythe/java/com/google/devtools/kythe/extractors/java/JavaCompilationUnitExtractor.java

```diff
@@ -128,6 +128,15 @@ private static String classJarRoot(Location location) {
   private final String rootDirectory;
   private final FileVNames fileVNames;
   private String systemDir;
+  private boolean allowServiceProcessors = true;
+
+  /**
+   * Set whether to allow service processors to be ran during extraction. Defaults to {@code true}.
```

Comment:
nit: "to run during extraction"

Reply:
Apparently auto-merge doesn't take into account comments.